### PR TITLE
WT-7542 Add a Python test to reconfigure zstd compression level after restart

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

File: test/suite/wttest.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ class WiredTigerTestCase(unittest.TestCase):
     @staticmethod
     def globalSetup(preserveFiles = False, removeAtStart = True, useTimestamp = False,
                     gdbSub = False, lldbSub = False, verbose = 1, builddir = None, dirarg = None,
-                    longtest = False, ignoreStdout = False, seedw = 0, seedz = 0, hookmgr = None):
+                    longtest = False, zstdtest = False, ignoreStdout = False, seedw = 0, seedz = 0, hookmgr = None):
```
Comment (lukech) [inline test/suite/wttest.py:206]: Should this new `zstdtest` argument be set in the function call in `test/suite/run.py`?
```
    wttest.WiredTigerTestCase.globalSetup(preserve, removeAtStart, timestamp, gdbSub, lldbSub,
                                          verbose, wt_builddir, dirarg, longtest,
                                          ignoreStdout, seedw, seedz, hookmgr)
```

**Review by lukech — CHANGES_REQUESTED** [review]
The direction of using a new command option to turn on zstd testing LGTM. Please see my comment about the setting of the new argument in the run.py function call. I think it's essential to make the new command option setting work. 

And I assume this is only a temporary solution, and once the zstd package is installed on PPC & zSeries machine images, you will reverse to your initial solution (i.e. removing the new command option and integrate your new Python test into unit-test). Could you open a follow-on ticket to address this work?

File: test/suite/wttest.py (hunk 1 above)
Comment (mm-ng) [reply test/suite/wttest.py:206]: Thanks Luke, I am going to add this argument now and then run another patch build.

**Review by lukech — APPROVED** [review]
LGTM

**Pull request merged**